BUILD.md

```diff
@@ -19,6 +23,7 @@ It may be easier to set up one over the other, depending on which flavour of Lin
 > ### Podman  
 >  
 > According to your distribution's particular package manager, install `podman`.  
+> `# xbps-install -S podman`  
```

Comment:
Probably not needed :) Void users can simply `xbps-install zfsbootmenu` to get it installed for them.

Reply:
:+1: